contracts/EverlongStrategy.sol

```diff
@@ -572,9 +774,30 @@ contract EverlongStrategy is BaseStrategy {
     ///         a new position.
     /// @return True if a new position can be opened, false otherwise.
     function canOpenPosition() public view returns (bool) {
-        return
-            asset.balanceOf(address(this)) >
-            _poolConfig.minimumTransactionAmount;
+        uint256 currentBalance = asset.balanceOf(address(this));
+        return currentBalance > _poolConfig.minimumTransactionAmount;
+    }
+
+    function convertToUnwrapped(
+        uint256 _wrappedAmount
+    ) public view returns (uint256 _unwrappedAmount) {
+        if (!isWrapped) {
+            revert IEverlongStrategy.AssetNotWrapped();
+        }
+        _unwrappedAmount = IHyperdrive(hyperdrive)._convertToBase(
+            _wrappedAmount
+        );
+    }
+
+    function convertToWrapped(
+        uint256 _unwrappedAmount
+    ) public view returns (uint256 _wrappedAmount) {
+        if (!isWrapped) {
+            revert IEverlongStrategy.AssetNotWrapped();
+        }
+        _wrappedAmount = IHyperdrive(hyperdrive)._convertToShares(
+            _unwrappedAmount
+        );
     }
```

Comment:
Nit: Let's add natspec to these functions.

Also, do we really need these? We can just use `convertToBase` for `convertToUnwrapped` and `convertToShares` for `convertToWrapped`. I'll leave this to you, but sticking with the old functions will reduce the lines of code.

Reply:
Good catch on the natspec here....

While I understand it introduces a LOC increase, I feel as though it makes it far less confusing to use this language as opposed to `convertToShares` / `convertToBase`.

Reply:
Good call, I agree after thinking more about it.